src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/TierToDeliusApiServiceTest.kt

```diff
@@ -45,70 +45,39 @@ class TierToDeliusApiServiceTest {
   }
 
   @Test
-  fun `Should return Breach points if false if present and valid terminationDate on cutoff`() = runBlocking {
-    val terminationDate = LocalDate.now().minusYears(1).minusDays(1)
-
-    val tierToDeliusResponse = TierToDeliusResponse(
-      "Male",
-      emptyList(),
-      listOf(DeliusConviction(terminationDate, sentenceTypeCode, true, emptyList())),
-      BigDecimal.TEN,
-      2,
-    )
-
-    coEvery { tierToDeliusApiClient.getDeliusTier(crn) } returns tierToDeliusResponse
-    val result = tierToDeliusApiService.getTierToDelius(crn).breached
-
-    Assertions.assertFalse(result)
-  }
-
-  @Test
-  fun `Should return Breach true if present and valid not terminated`() = runBlocking {
-    val tierToDeliusResponse = TierToDeliusResponse(
-      "Male",
-      emptyList(),
-      listOf(DeliusConviction(null, sentenceTypeCode, true, emptyList())),
-      BigDecimal.TEN,
-      2,
-    )
-
-    coEvery { tierToDeliusApiClient.getDeliusTier(crn) } returns tierToDeliusResponse
-    val result = tierToDeliusApiService.getTierToDelius(crn).breached
-    Assertions.assertTrue(result)
-  }
-
-  @Test
-  fun `Should return Breach true if multiple convictions, one valid`() = runBlocking {
+  fun `Previous Enforcement Activity Should be true if API response is true`() = runBlocking {
     val tierToDeliusResponse = TierToDeliusResponse(
       "Male",
       emptyList(),
-      listOf(
-        DeliusConviction(null, sentenceTypeCode, true, emptyList()),
-        DeliusConviction(null, sentenceTypeCode, false, emptyList()),
-      ),
+      listOf(DeliusConviction(null, sentenceTypeCode, emptyList())),
       BigDecimal.TEN,
       2,
+      true,
     )
 
     coEvery { tierToDeliusApiClient.getDeliusTier(crn) } returns tierToDeliusResponse
-    val result = tierToDeliusApiService.getTierToDelius(crn).breached
-    Assertions.assertTrue(result)
+    val result = tierToDeliusApiService.getTierToDelius(crn)
+    Assertions.assertTrue(result.previousEnforcementActivity)
+    Assertions.assertTrue(result.rsrScore == BigDecimal.TEN)
```

Comment:
Suggest to remove the assertions which are not related to the test description and test those separately